packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -12,6 +12,11 @@ export interface RadioGroupState extends RadioGroupProps {
    */
   value: string | null;
 
+  /**
+   * Layout of the Radios. True if horizontal or horizontal-stacked.
+   */
+  isHorizontal?: boolean;
```

Comment:
Is this needed if RadioGroupState extends RadioGroupProps and it already has `layout`?

Reply:
I guess it helps with token setting.

Reply:
Though the logic could be moved to the lookup function without needing to add a property to the state I think

Reply:
Moved the logic to lookup function, thanks!